lib/Transport.js

```diff
@@ -340,7 +340,18 @@ module.exports = class Transport
 
   _onData(data)
   {
-    // CRLF Keep Alive response from server. Ignore it.
+    // CRLF Keep Alive request from server, reply.
+    if (data === '\r\n\r\n')
+    {
+      logger.debug('received message with double-CRLF Keep Alive request');
+
+      // Reply with single CRLF.
+      this.socket.send('\r\n');
```

Comment:
I'd add a catch here (not sure if try/catch or .catch()) since socket.send() can throw (or reject?).